src/http.h

```diff
@@ -18,3 +18,8 @@ struct _aws_crt_http_message {
     struct aws_http_message *message;
     struct aws_byte_buf encoded_message;
 };
+
+struct _aws_crt_buf {
+    struct aws_http_headers *headers;
+    struct aws_byte_buf encoded_headers;
+};
```

Comment:
Not sure what this is for?

Reply:
oops I thought I deleted it but apparently I did not